file/file_test.go

```diff
@@ -22,139 +21,28 @@ var (
 	//
 	// see:
 	// https://docs.minio.io/docs/golang-client-api-reference
-	testEndpoint  = "play.minio.io:9000"
-	testAccessKey = "Q3AM3UQ867SPQQA43P2F"
-	testSecretKey = "zuf+tfteSlswRu7BJ86wekitnifILbZam1KYY3TG"
-	testBucket    = "task-tools-test"
-	testS3Client  *minio.Client
-)
-
-func TestGlob_Local(t *testing.T) {
-	// create files
-	pths := []string{
-		"./test/file-1.txt",
-		"./test/file2.txt",
-		"./test/file3.gz",
-	}
-	for _, pth := range pths {
-		createFile(pth)
-	}
-
-	// test '*'
-	pth := "./test/*.txt"
-	allSts, err := Glob(pth, nil)
-	if err != nil {
-		t.Errorf("expected nil but got '%v'\n", err.Error())
-	}
-
-	if len(allSts) != 2 {
-		t.Errorf("expected %v got %v\n", 2, len(allSts))
-	}
-
-	// test '?'
-	pth = "./test/file?.txt"
-	allSts, err = Glob(pth, nil)
-	if err != nil {
-		t.Errorf("expected nil but got '%v'\n", err.Error())
-	}
-
-	if len(allSts) == 1 {
-		f1 := allSts[0]
-		expected := "file2.txt"
-		_, got := path.Split(f1.Path)
-		if got != expected {
-			t.Errorf("expected file '%v' got '%v'\n", expected, got)
-		}
-	} else {
-		t.Errorf("expected %v got %v\n", 1, len(allSts))
-	}
-
-	// test '?' with '*'
-	pth = "./test/file?.*"
-	allSts, err = Glob(pth, nil)
-	if err != nil {
-		t.Errorf("expected nil but got '%v'\n", err.Error())
-	}
-
-	if len(allSts) != 2 {
-		t.Errorf("expected %v got %v\n", 2, len(allSts))
-	}
-
-	// cleanup
-	for _, pth := range pths {
-		rmFile(pth)
-	}
-	rmFile("./test")
-}
-
-func TestGlob_S3(t *testing.T) {
-	// create files
-	opt := NewOptions()
-	opt.AccessKey = testAccessKey
-	opt.SecretKey = testSecretKey
-	pthDir := fmt.Sprintf("s3://%v/test", testBucket)
-	pths := []string{
-		fmt.Sprintf("%v/file-1.txt", pthDir),
-		fmt.Sprintf("%v/file2.txt", pthDir),
-		fmt.Sprintf("%v/file3.gz", pthDir),
-	}
-	for _, pth := range pths {
-		createFile(pth)
-	}
-
-	// test '*'
-	pth := fmt.Sprintf("%v/*.txt", pthDir)
-	allSts, err := Glob(pth, opt)
-	if err != nil {
-		t.Errorf("expected nil but got '%v'\n", err.Error())
-	}
-
-	if len(allSts) != 2 {
-		t.Errorf("expected %v got %v\n", 2, len(allSts))
-	}
-
-	// test '?'
-	pth = fmt.Sprintf("%v/file?.txt", pthDir)
-	allSts, err = Glob(pth, opt)
-	if err != nil {
-		t.Errorf("expected nil but got '%v'\n", err.Error())
-	}
+	testEndpoint = "play.minio.io:9000"
+	testBucket   = "task-tools-test"
+	testS3Client *minio.Client
 
-	if len(allSts) == 1 {
-		f1 := allSts[0]
-		expected := "file2.txt"
-		_, got := path.Split(f1.Path)
-		if got != expected {
-			t.Errorf("expected file '%v' got '%v'\n", expected, got)
-		}
-	} else {
-		t.Errorf("expected %v got %v\n", 1, len(allSts))
+	wd   string
+	opts = Options{
+		AccessKey: "Q3AM3UQ867SPQQA43P2F",
+		SecretKey: "zuf+tfteSlswRu7BJ86wekitnifILbZam1KYY3TG",
```

Comment:
is having the secrets in here a concern?

Reply:
So these "secrets" are the free to use test credentials to connect to minio which we use to simulate aws. The are provided on the open web so they aren't really a secret. I'd like to stop using that and have minio running in a docker image as part of circleci, but haven't gotten around to it  yet.